### DISCUSSION TITLE
Secondary Monitor Support

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#16403] [Feature requests]
A simple option to switch to secondary monitor with the OpenGL renderer would be nice. Some games let you change monitor id from monitor 0 to monitor 1. I would  be willing to help the developers test the feature if they don't have multiple monitor setups
![2022-01-16](https://user-images.githubusercontent.com/94884086/149654492-590c2244-373e-4a3e-aad7-a43c848a149c.png)

**Comment by IntelOrca** [comment]
Does switching to window mode, moving the window to the desired monitor and then switching back to fullscreen work?

**Reply by 733737** [reply]
I have to use dual monitor tools hotkeys but once I get it to the second monitor it will stay there "forever"

**Reply by 733737** [reply]
to be more specific, it is not easy. So you cannot simply drag it over and resize it, no.

**Reply by ZehMatt** [reply]
Any particular reason you can't use the Fullscreen borderless mode? It is pretty much the same and you can just move it on any desktop you like.

**Reply by 733737** [reply]
I actually have window snapping turned off

**Comment by ocalhoun6** [comment]
This is probably _way_ outside the scope of this discussion, but ...

What if OpenRCT2 could actually _make use_ of multiple monitors? 

It would be pretty amazing to have the main park open on one monitor, with the mini-map open on the other. Maybe also ride construction windows or any other kind of open window, even if the main park viewport can't be spanned across the multiple screens.

**Reply by pizza2004** [reply]
This is my dream.  I salivate over the mere idea of games with multiple monitor support.  You could have a different viewport on each monitor and just quickly look back and forth.  It would be incredible.

**Reply by ZehMatt** [reply]
The reason barely any game supports this is probably due to the technical debt to do it, so this is very unlikely to ever be implemented.

**Reply by 733737** [reply]
Instead of pie in the sky requests we should keep the ask to a limited and specific

 thing, namely what's referenced in this screencap ![2022-01-16](https://user-images.githubusercontent.com/94884086/149654492-590c2244-373e-4a3e-aad7-a43c848a149c.png)
